ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,278 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  /**
+   * Common assertions for Add terms, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
+   * @param {Object} component - Root DOM object in react class
+   * @return {Object} devItems - DOM object of class 'panel-body'
```

Comment:
`Object` here should be changed to `ReactComponent` also, due to the return type for `findRenderedDOMComponentWithClass` listed in the React docs.